source/recall.js

```diff
@@ -0,0 +1,35 @@
+import uncurryN from './uncurryN';
+
+/**
+ * Collapses a high-order unary function. The order N of the function
+ * is implicit. A maximum number of N values are singularly fed.
+ *
+ * @func
+ * @category Function
+ * @sig ((a -> b -> * -> c), *...) -> d
+ * @param {Function} fn The N-order function to be collapsed.
+ * @param {...*} args Up to N input values.
+ * @return {*}
+ * @see R.call
+ * @example
+ *
+ *      const xs = [{a: 1}, {a: 2}, {a: 3}];
+ *      const f = R.compose(R.find, R.propEq('a'));
+ *      R.recall(f, 2, xs); //=> {"a": 2}
+ *
+ * @symb R.recall(f, a, b) = f(a)(b)
+ */
+var recall = uncurryN(2, function(fn) {
```

Comment:
I'm extremely loathe to add new variadic functions to Ramda.

I would much rather deal with an API such as this:

```js
recall (f) (a, b) //=> f (a) (b)
```

(Even that probably isn't possible.  I'm quite certain that we will need to supply the final arity.  But that's a different comment.)

I know that the generated function is going to be polyadic.  That's the whole point.  And it's not even as though it always returns functions of a specific arity.  But I'm hoping to make `pipe`, `compose` and other leftover variadic functions into fixed-arity ones.

Reply:
I feel the same way, I was just mirroring `call`. The list of values can be passed as an array instead, as in `apply`.

Reply:
Yeah, the `call` API was an early mistake as we felt our way around this space.

Reply:
The idea is to call a function with a set of values in the fashion of `call` and `apply`. The difference here is there is no notion of arity since the n-order function is composed of unary functions. The list of values supplied doesn't represent an arity of a function, rather a serial list of singular inputs fed to the n-order function. So returning a function defeats the purpose in which case I'd rather just use `uncurryN` somehow.

Reply:
Perhaps then we're at cross purposes.  I was thinking that the goal was a to bring behavior similar to `uncurryN` to work with Ramda-styles curried functions. I thought we were mostly just removing the word "manually" from the docs.

I'm not sure what your goal is hers.

Reply:
Ideally `uncurryN` would be made to work but I'd still want this functionality, otherwise I'd have just submitted a patch. Is anyone looking into it?

Reply:
I don't think so.  I've been pulled into something else, and haven't had any real free time for Ramda in the past week.  If no one does it before then, I will try to get to it this weekend.  (You can do it yourself.  Any conversations of importance about this would be in the issues here; so a GitHub search should probably show anything of importance that isn't already captured in the source or the tests.)